src/pages/Create/Create.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { ChevronDown } from 'lucide-react';
+import { mockDecks } from '../../data/mockDecks';
+
+const Create = () => {
+  const [selectedDeck, setSelectedDeck] = useState('');
+  const [frontContent, setFrontContent] = useState('');
+  const [backContent, setBackContent] = useState('');
+
+  return (
+    <div className="p-4 max-w-xl mx-auto">
+      {/* Zaglavlje */}
+      <h1 className="text-2xl font-bold mb-6">Create New Card</h1>
+
+      <form className="space-y-6">
+        {/* Odabir špila */}
+        <div className="space-y-2">
+          <label className="block text-sm font-medium">
```

Comment:
if we have a label separated from input/select field, we need htmlFor attribute on label element and then id attribute on input/select element